lib/membrane_wav/serializer_bin.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.WAV.SerializerBin do
```

Comment:
After the suggested changes this module won't be much of a help, especially since it adds a dependency on `file_plugin`. Of course, we could do some magic here and make `file_plugin` dependency optional (I had in mind a runtime check on `File.Sink` module existence, raising an error if it's missing and instructing to add `file_plugin` do deps, something similar to what is done for example in Tesla adapters)

TL;DR This can be removed, sorry